admin-dev/themes/default/js/tree.js

```diff
@@ -213,13 +210,13 @@ Tree.prototype =
 
 			$.get(
 				'ajax-tab.php',
-				{controller:'AdminProducts',token:currentToken,action:'getCategoryTree',type:idTree,fullTree:1,selected:selected, inputName:name,useCheckBox:useCheckBox},
+				{ controller:'AdminProducts', token:currentToken, action:'getCategoryTree', type:idTree, fullTree:1, selected:selected, inputName:name, useCheckBox:useCheckBox },
 				function(content)
 				{
 					$('#' + idTree).html(content);
 					organizeTree();
 					$('#' + idTree).tree('init');
-					that.$element.find("label.tree-toggler").each(
+					this.$element.find("label.tree-toggler").each(
```

Comment:
What's the reference of `this` there ? It looks like it's scope is the anonymous function

Reply:
This is a mistake it should be $(this)

The issue is that the current code uses variables for $(this) 
`var that = $(this);` and `var thatOne = $(this);`

However these variables are not passed in the functions `function(content)` which causes the error.

Reply:
that wasn't accessible ? It looks like it's in the upperscope, it should be accessible ! strange thing there